site/sfguides/src/servicenow_to_snowflake_connector/servicenow_to_snowflake_connector.md

```diff
@@ -338,20 +336,21 @@ During this lab, we're only ingesting the data, so it makes sense to stop the in
 
 1. In Snowsight, select the **Snowflake Connector for ServiceNow** tile.
 
-1. In the **Snowflake Connector for ServiceNow** window, select **Stop Ingestion**.
+1. In the **Snowflake Connector for ServiceNow** window, select **Pause Connector**.
 
-![stop](assets/stop.png)
+## Uninstall the Connector (but not the data)
+Duration: 1
 
-Read the warning and select **Stop Ingestion**.
+If you completed the experiment or for any reason no longer need the connector you can easily uninstall it via the Snowflake Marketplace.
 
-## Uninstall the Connector (but not the data)
-If you are using the public preview connector, make sure to check out the limitations, one of which is during the preview period, before the connector is generally available, Snowflake will release an update that requires you to **uninstall** and reinstall the connector from the Snowflake Marketplace.
+1. Select **Data Products** and then **Apps**.
+1. Select three dots icon in the item on the list representing the connector app.
+1. Select **Uninstall**
+1. Decide if you want to delete the objects owned by the application (tables and views in the destination schema) or transfer ownership of them to another role
```

Comment:
Nit: maybe it's worth to stress more that objects owned by the application include ingested data, e.g.: Decide if you want to delete the objects owned by the application (including tables and views with ingested data in the destination schema) or transfer ownership of them to another role

Reply:
added